src/Mjolnir.ts

```diff
@@ -399,13 +399,40 @@ export class Mjolnir {
         }
     }
 
+    public async getProtectionSettings(protectionName: string): Promise<{ [setting: string]: any }> {
+        const eventName = `org.matrix.mjolnir.set.${protectionName}`;
+        return await this.client.getRoomStateEvent(config.managementRoom, eventName, "").catch(err => {
+        }) || {};
+    }
+    public async setProtectionSettings(protectionName: string, changedSettings: { [setting: string]: any }): Promise<any> {
+        const eventName = `org.matrix.mjolnir.set.${protectionName}`;
+        const settings = await this.getProtectionSettings(protectionName);
+
+        const combo = {
+            ...settings,
```

Comment:
Would it be useful to validate this data?
Or is this the low-level storage layer on top of which validation should be implemented?